src/sql/parts/notebook/notebookInput.ts

```diff
@@ -28,11 +28,13 @@ export class NotebookInputModel extends EditorModel {
 	private _providerId: string;
 	private _standardKernels: IStandardKernelWithProvider[];
 	private _defaultKernel: sqlops.nb.IKernelSpec;
-	constructor(public readonly notebookUri: URI, private readonly handle: number, private _isTrusted: boolean = false, private saveHandler?: ModeViewSaveHandler, provider?: string, private _providers?: string[]) {
+	private _connectionProfileId: string;
+	constructor(public readonly notebookUri: URI, private readonly handle: number, private _isTrusted: boolean = false, private saveHandler?: ModeViewSaveHandler, provider?: string, private _providers?: string[], connectionProfileId?: string) {
```

Comment:
Why not defined the private field in the constructor like the other parameters do? Also, I think we've got enough params where we should now do the "one line per param" solution:
```suggestion
	constructor(public readonly notebookUri: URI,
		private readonly handle: number,
		private _isTrusted: boolean = false,
		private saveHandler?: ModeViewSaveHandler,
		provider?: string,
		private _providers?: string[],
		private _connectionProfileId?: string) {
```